app/src/main/java/org/wikipedia/readinglist/ReadingListActivity.kt

```diff
@@ -29,6 +32,13 @@ class ReadingListActivity : SingleFragmentActivity<ReadingListFragment>() {
         setNavigationBarColor(getThemedColor(this, R.attr.paper_color))
     }
 
+    override fun onBackPressed() {
+        super.onBackPressed()
+        if (isPreview) {
```

Comment:
You can do the check for `intent.getBooleanExtra(...)` right here, and not need to keep it in a class field.

Reply:
Done @dbrant